packages/less/benchmark/benchmark-runner.js

```
@@ -120,6 +155,41 @@ function runAll(i) {
+        max = Math.max(max, effective[i]);
+    }
+    var avg = total / effective.length;
+
+    // Median
+    var sorted = effective.slice().sort(function (a, b) { return a - b; });
+    var mid = Math.floor(sorted.length / 2);
+    var median = sorted.length % 2 ? sorted[mid] : (sorted[mid - 1] + sorted[mid]) / 2;
+
+    // Standard deviation and coefficient of variation
+    var sumSqDiff = 0;
+    for (var i = 0; i < effective.length; i++) {
```

ESLint

[error] 'i' is already defined.

(no-redeclare)